scripts/ocr_runner.py

```
@@ -0,0 +1,198 @@
+        out_pdf = OCR_DIR / f'{sha}.ocr.pdf'
+        try:
+            ret = run_ocrmypdf(path, out_pdf, logfile, ocr_timeout)
+            rec['ocr_return'] = ret
+            if ret != 0:
+                rec['ocr_status'] = 'ocr_failed'
+            else:
+                rec['after_chars'] = pdftotext_count(out_pdf)
+                rec['ocr_status'] = 'ok' if rec['after_chars'] > 0 else 'ocr_empty'
+        except subprocess.TimeoutExpired:
+            rec['ocr_status'] = 'ocr_timeout'
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)